python/lsst/meas/base/references.py

```diff
@@ -183,8 +197,8 @@ def getWcs(self, dataRef):
 
     def fetchInPatches(self, dataRef, patchList):
         """!
-        An implementation of BaseReferencesTask.fetchInPatches that loads 'coadd_src' catalogs
-        using the butler.
+        An implementation of BaseReferencesTask.fetchInPatches that loads 'coadd_datasetSuffix'
+        catalogs using the butler.
```

Comment:
Maybe set off the "datasetSuffix" string somehow, so it's clear it's a variable, not a literal?